app/models/Actionlog.php

```diff
@@ -1,85 +1,193 @@
 
-    public function adminlog()
-    {
-        return $this->belongsTo('User','user_id')->withTrashed();
-    }
+        public function licenselog()
+        {
 
-    public function userlog()
-    {
-        return $this->belongsTo('User','checkedout_to')->withTrashed();
-    }
+            return $this->belongsTo( 'License', 'asset_id' )
+                        ->withTrashed();
+        }
 
-    public function childlogs()
-    {
+        public function accessorylog()
+        {
 
-        return $this->hasMany( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Accessory', 'accessory_id' )
+                        ->withTrashed();
+        }
 
-    public function parentlog()
-    {
+        public function consumablelog()
+        {
 
-        return $this->belongsTo( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Consumable', 'consumable_id' )
+                        ->withTrashed();
+        }
 
-    /**
-	* Check if the file exists, and if it does, force a download
-	**/
-    public function get_src() {
+        public function adminlog()
+        {
 
-			$file = app_path().'/private_uploads/'.$this->filename;
-			return $file;
+            return $this->belongsTo( 'User', 'user_id' )
+                        ->withTrashed();
+        }
 
-    }
+        public function userlog()
+        {
 
+            return $this->belongsTo( 'User', 'checkedout_to' )
+                        ->withTrashed();
+        }
 
+        public function childlogs()
+        {
 
-    /**
-    * Get the parent category name
-    */
-    public function logaction($actiontype)
-    {
-        $this->action_type = $actiontype;
+            return $this->hasMany( 'ActionLog', 'thread_id' );
+        }
 
-        if($this->save()) {
-            return true;
-        } else {
-            return false;
+        public function parentlog()
+        {
+
+            return $this->belongsTo( 'ActionLog', 'thread_id' );
+        }
+
+        /**
+         * Check if the file exists, and if it does, force a download
+         **/
+        public function get_src()
+        {
+
+            $file = app_path() . '/private_uploads/' . $this->filename;
+
+            return $file;
+
+        }
+
+        /**
+         * Get the parent category name
+         */
+        public function logaction( $actiontype )
+        {
+
+            $this->action_type = $actiontype;
+
+            if ($this->save()) {
+                return true;
+            } else {
+                return false;
+            }
         }
-    }
-}
+
+        /**
+         * getUnacceptedAssets
+         * @param array $listOfAssets
+         *
+         * @return mixed
+         * @static
+         * @author  Vincent Sposato <vincent.sposato@gmail.com>
+         * @version v1.0
+         */
+        public static function getUnacceptedAssets( $listOfAssets = [ '*' ] )
+        {
+
+            $assetIds = implode( ",", $listOfAssets );
```

Comment:
That should still take prefixes into consideration, and even then, we're not using any in the travis build. Plus the table name in the error isn't showing one. In the trace from the failed build, it does point to the new artisan commands you created (why did you do those as artisan commands instead of a migration btw?) but the errors are not suggesting table prefixes as being the issue.